spec/features/front_end_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "spec_helper"
 
-RSpec.describe "Front end" do
+RSpec.fdescribe "Front end" do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/MultipleMemoizedHelpers: Example group has too many memoized helpers [9/5]

Reply:
In other project we disabled this rule... what should I do here?